app/routes/dog_routes.py

```diff
@@ -8,12 +8,12 @@ def __init__(self, id, name, breed, tricks=None):
         self.name = name
         self.breed = breed
         if not tricks:
-            tricks = []
+            self.tricks = []
         self.tricks = tricks
 
     def to_json(self):
         if not self.tricks:
-            tricks = "No Tricks"
+            tricks = []
         else:
```

Comment:
This isn't necessary to change now, but because of the init, self.tricks should never be `None`, is the if/else really needed here?